ui/static/ui/js/static_assets.jsx

```diff
@@ -1,8 +1,15 @@
 define('static_assets', [
-  'react', 'jquery', 'lodash', 'utils'], function (React, $, _, Utils) {
+  'react', 'jquery', 'lodash', 'utils'
+  ], function (React, $, _, Utils) {
   'use strict';
 
   var StatusBox = Utils.StatusBox;
+  var InfiniteList = Utils.InfiniteList;
+
+  // Guess at a good value for the height of each element. Infinite requires
+  // an elementHeight for each item. If we wanted to do a per item elementHeight
+  // we could make this a list instead.
+  var elementHeight = 50;
```

Comment:
Everything I've tested behaves the same. 

I will probably run this UI by the designer. I think there's probably some polish we can add, but it totally meets the needs of the story.